tools/c7n_tencentcloud/c7n_tencentcloud/resources/cdb.py

```diff
@@ -0,0 +1,41 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod, isoformat_datetime_str
+import pytz
+
+
+@resources.register("cdb")
+class CDB(QueryResourceManager):
+    """cdb"""
```

Comment:
please put the full name of the service here, the acronyms aren't clearly obvious to, even then I wonder if we just want to have the type named db-instance here for example to make it more obvious.

Reply:
optimized